projects/BFDR/BirthRecord_submissionProperties.cs

```diff
@@ -5437,5 +5437,294 @@ public string PayorTypeFinancialClassHelper
                 }
             }
         }
+
+        /// <summary>Set an emerging issue value, creating an empty EmergingIssues Observation as needed.</summary>
+        private void SetEmergingIssue(string identifier, string value)
+        {
+            Observation EmergingIssues = CreateObservationEntry(identifier, Child.Id, EMERGING_ISSUES_SECTION);
```

Comment:
there was a slight refactor to this function that was just merged to Natality-feature. The helper name changed to GetOrCreateObservation and it takes a few more optional parameters to create the observation. Rebase on Natality-feature and update this to use the new helper